CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
+* [#](https://github.com/nf-core/ampliseq/pull/) - Make `--skip_dada_addspecies` compatible with `--cut_its`
 * [#352](https://github.com/nf-core/ampliseq/pull/352) - `--skip_dada_addspecies` allows to skip species level classification to reduce memory requirements, incompatible with `--sbdiexport` and `--cut_its` that expect species annotation
```

Comment:
Changes are usually relative to the last release, so those two points could be merged, i.e.
```suggestion
* [#352](https://github.com/nf-core/ampliseq/pull/352), [#372])https://github.com/nf-core/ampliseq/pull/372) - `--skip_dada_addspecies` allows to skip species level classification to reduce memory requirements, incompatible with `--sbdiexport` and `--cut_its` that expect species annotation
```

Reply:
Good point, I'll fix that.